src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -686,6 +696,15 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.accessKeySortDirection = 1;
       /** @type {(msgId: string, ...params: string[]) => string} */
       this.localize = null;
+      this.currentPage = '';
+      this.selectedTab = 'connections';
+    }
+
+    selectPage(page) {
+      if (page !== 'progressView') {
+        this.$.progressView.stop();
+      }
+      this.currentPage = page;
```

Comment:
I think you are no longer starting the animation. Perhaps you should do that on selected-changed too.